gocd/templates/clickhouse-query-comparer.jsonnet

```diff
@@ -0,0 +1,67 @@
+local gocdtasks = import 'github.com/getsentry/gocd-jsonnet/libs/gocd-tasks.libsonnet';
+
+local pipeline_group = 'snuba';
+local pipeline_name = 'clickhouse-query-fetcher';
+
+local generate_compare_job() =
+  {
+    environment_variables: {
+      SENTRY_REGION: 's4s',
+      SNUBA_CMD_TYPE: 'comparer',
+    },
+    elastic_profile_id: pipeline_group,
+    tasks: [
+      gocdtasks.script(importstr './bash/s4s-clickhouse-queries.sh'),
+    ],
+  };
+
+local pipeline = {
+  environment_variables: {
```

Comment:
You want all the arguments to be set in gocd under here. I'm assuming that would be QUERYLOG_HOST e.t.c

Reply:
@dbanda I was only putting the args that are needed for the comparer here - since we don't need the other args